.github/workflows/sync.yml

```
@@ -118,57 +124,102 @@ jobs:
+          echo "✅ Rama $SYNC_BRANCH creada y enviada a origin"
+          echo "branch_name=$SYNC_BRANCH" >> $GITHUB_OUTPUT
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # Ahora usamos la CLI de GitHub para crear el PR directamente
+      - name: Crear PR con GitHub CLI
+        id: create_pr
+        if: steps.detect_hotfix.outputs.is_hotfix == 'true' && steps.prepare.outputs.develop_exists == 'true' && steps.prepare.outputs.need_sync == 'true' && steps.create_branch.outputs.branch_name != ''
+        run: |
+          SYNC_BRANCH="${{ steps.create_branch.outputs.branch_name }}"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)